romancal/pipeline/exposure_pipeline.py

```diff
@@ -173,10 +173,6 @@ def process(self, input):
                 result.meta.cal_step.tweakreg = "SKIPPED"
                 self.suffix = "cal"
```

Comment:
I think it's safe to remove this from here, since we're calling `self.setup_output()` right after the `if` statement block. Also, I believe we'll be implementing a way to set the proper suffix based on the input passed on to that method, correct?